projects/storefrontlib/shared/components/chat-messaging/messaging/messaging.model.ts

```diff
@@ -0,0 +1,26 @@
+export interface MessageEvent {
+  author?: string;
+  rightAlign?: boolean;
+  createdAt?: string;
+  text?: string;
+  attachments?: Array<Attachment>;
+}
+
+export interface Attachment {
+  filename: string;
+  URL: string;
```

Comment:
any reason why this is not lowercase like other attributes?

Reply:
this needs to be replaced with `id`